operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md

```diff
@@ -49,6 +49,10 @@ For more detail about about the CSM upgrade hooks, see the section [description
 
    Follow the IUF [Product delivery](product_delivery.md) instructions.
 
+   SMA 1.10.15 and later includes an upgraded LDMS that introduces an incompatibility with configuration files used in prior versions.
+    * When upgrading from an older SMA version to a version with this new LDMS, the administrator must change the configuration files.
+    * A workaround is presented as an Action in the deliver-product stage in the **IUF Stage Details for SMA** section of the _HPE Cray Supercomputing EX System Monitoring Application Installation Guide_.
```

Comment:
This is changing this pages unordered list identifier to `*` and causing the lists further down the page to now fail the stylechecker. https://github.com/Cray-HPE/docs-csm/actions/runs/14227293097/job/39869738860?pr=5865

Please use `-` or change the remaining lists to use `*`.